app/leaderboard/_components/LeaderboardWrapper.tsx

```diff
@@ -16,7 +17,7 @@ export default async function LeaderboardWrapper({
       | "intern"
       | "operations"
       | "contributor"
-    )[]) ?? [],
+    )[]) ?? deaultRoles.map((i) => i.value),
```

Comment:
Let's keep this configurable as different organizations require different behaviour. Move this to the environment: `NEXT_PUBLIC_LEADERBOARD_DEFAULT_ROLES=comma,seperated,values`